src/scc_hypervisor_collector/cli/__init__.py

```diff
@@ -38,8 +82,14 @@ def main(argv: Optional[Sequence[str]] = None) -> None:
                         help="The config directory to check for YAML "
                              "config files.")
     parser.add_argument('-C', '--check', action='store_true',
-                        help="Check the configuration data only, "
-                             "reporting any errors.")
+                        help="Check the configuration data "
+                             "only reporting any errors.")
```

Comment:
Lost the `,` after `only`.

```suggestion
                             "only, reporting any errors.")
```